README.md

```diff
@@ -47,6 +80,20 @@ docker compose down --volume
 docker compose logs --follow --since=5m
 ```
 
+## Using environment variables
+
+### Setup `mise` on Windows
+
+_Instructions to come_.
```

Comment:
We needed to add mise to the windows path. The value to add to the path was: `%USERPROFILE%\AppData\Local\mise\shims`. 

Run `mise doctor` after running `choco install <config-file>` in an administrator powershell session.

The docs also need to be updated to include these steps.